src/test/java/net/sf/jabref/importer/fileformat/InspecImportTest.java

```diff
@@ -0,0 +1,144 @@
+                .getResourceAsStream("IsiImporterTestWOS.isi"); InputStream stream6 = InspecImportTest.class
+                .getResourceAsStream("IsiImporterTestMedline.isi"); InputStream stream7 = InspecImportTest.class
+                .getResourceAsStream("InspecImportTest.txt"); InputStream stream8 = InspecImportTest.class
+                .getResourceAsStream("InspecImportTest2.txt"); InputStream stream9 = InspecImportTest.class
+                .getResourceAsStream("InspecImportTest3.txt"); InputStream stream10 = InspecImportTest.class
+                .getResourceAsStream("InspecImportTest4.txt"); InputStream stream11 = InspecImportTest.class
+                .getResourceAsStream("InspecImportTestFalse.txt")){
+
+            assertFalse(inspecImp.isRecognizedFormat(stream1));
+
+            assertFalse(inspecImp.isRecognizedFormat(stream2));
+
+            assertFalse(inspecImp.isRecognizedFormat(stream3));
+
+            assertFalse(inspecImp.isRecognizedFormat(stream4));
+
+            assertFalse(inspecImp.isRecognizedFormat(stream5));
+
+            assertFalse(inspecImp.isRecognizedFormat(stream6));
+
+            assertTrue(inspecImp.isRecognizedFormat(stream7));
+
+            assertTrue(inspecImp.isRecognizedFormat(stream8));
+
+            assertTrue(inspecImp.isRecognizedFormat(stream9));
+
+            assertTrue(inspecImp.isRecognizedFormat(stream10));
+
+            assertFalse(inspecImp.isRecognizedFormat(stream11));
+        }
+
+        InputStream inStream = InspecImportTest.class.getResourceAsStream("InspecImportTest.txt");
+        InputStream inStreamFalse = InspecImportTest.class.getResourceAsStream("InspecImportTestFalse.txt");
+
+        Assert.assertTrue("InspecImporter isRecognizedFormaTrue test failed",
+                inspecImp.isRecognizedFormat(inStream));
+        Assert.assertFalse("InspecImporter isRecognizedFormatFalse test failed",
+                inspecImp.isRecognizedFormat(inStream));
+
+    }
+
+    @Test
+    public void testImportEntries1() throws IOException{
+
+
+        try(InputStream inStream = InspecImportTest.class.getResourceAsStream("InspecImportTest2.txt")) {
+            List<BibtexEntry> entries = inspecImp.importEntries(inStream, new OutputPrinterToNull());
+            assertEquals(1, entries.size());
+            BibtexEntry entry = entries.get(0);
+
+            assertEquals("The SIS project : software reuse with a natural language approach", entry.getField("title"));
+            assertEquals("Prechelt, Lutz", entry.getField("author"));
+            assertEquals(" 1992", entry.getField("year"));
+            assertEquals("Abstrakt", entry.getField("abstract"));
+            assertEquals("key", entry.getField("keywords"));
+            assertEquals("10000", entry.getField("journal"));
+            assertEquals("20", entry.getField("pages"));
+            assertEquals(" 19", entry.getField("volume"));
```

Comment:
In the beginning i though we weren't allowed to change the Import classes, that's why i created the tes tlike this, I'll fix that in the next commit, so that it strips the white spaces off.